src/index.css

```diff
@@ -44,7 +44,34 @@
   font-weight: 600;
 }
 
+.navbar > span {
+  width: 40px;
+  height: 40px;
+  position: relative;
+  left: 99%;
+  margin-top: 40px;
+  border-radius: 50%;
+  border: 1.5px solid #888;
+}
+
+.profile {
+  position: relative;
+  left: 12px;
+  top: 12px;
+  color: blue;
+}
+
 /* books styling */
+
+.chart {
+  width: 92%;
+  position: relative;
+  left: 3%;
+  border-radius: 10px;
+  height: 2px;
+  background-color: rgb(255, 255, 255);
+}
+
```

Comment:
- [ ] As  required for the project, kindly make sure your styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) e.g.: Text colors, Fonts(size, weight, family,...), background-colors, alignment, spscing(psdding & mergin),...
### Template design
![Screenshot from 2023-04-06 19-22-04](https://user-images.githubusercontent.com/83042742/230512111-d4d9ed9c-f465-4f85-b8f5-703b07f8e5fb.png)

### Your Design
![Screenshot from 2023-04-06 19-27-39](https://user-images.githubusercontent.com/83042742/230512134-c9dca661-7b6b-49af-b565-3ec2807dbee3.png)